examples/control_flow/hybrid_control_flow/src/nat_hybrid_control_flow/register.py

```
@@ -0,0 +1,139 @@
+from nat.cli.register_workflow import register_function
+from nat.data_models.function import FunctionBaseConfig
+
+logger = logging.getLogger(__name__)
+
+
+class MockInputValidatorFunctionConfig(FunctionBaseConfig, name="mock_input_validator"):
+    pass
+
+
+@register_function(config_type=MockInputValidatorFunctionConfig, framework_wrappers=[LLMFrameworkEnum.LANGCHAIN])
+async def mock_input_validator_function(config: MockInputValidatorFunctionConfig, builder: Builder):
```

Ruff (0.14.14)

[warning] Unused function argument: `config`

(ARG001)